### PR TITLE
feat(MOSIP-29236): add reusable workflows

### PR DESCRIPTION
**Pull request by adityankannan-tw**

### CODE REVIEW

File: android/fastlane/Fastfile [hunk 1]
```diff
@@ -36,15 +41,16 @@ lane :android_build_internal do
...
   upload_to_play_store(
   track: 'internal',
   release_status: 'completed',
   version_name: versionName,
+  package_name: applicationId,
   )
 
   slack(
-      message: "Inji - #{versionName} (#{current_build_number}) is uploaded to Play store. Description : #{PLAY_CONSOLE_RELEASE_DESCRIPTION}.", 
+      message: "Inji - #{INJI_FLAVOR} Wallet- #{versionName} (#{current_build_number}) is uploaded to Play store. Description : #{PLAY_CONSOLE_RELEASE_DESCRIPTION}.",
```
Comment (swatigoel) [inline android/fastlane/Fastfile:53]: Can we use app_name created for each flavor in build gradle

File: android/fastlane/Fastfile [hunk 2]
```diff
@@ -71,15 +77,15 @@ lane :android_build_beta do
...
+  gradle(task: "clean bundle#{INJI_FLAVOR}Release")
   upload_to_play_store(
   track: 'alpha',
   release_status: 'completed',
   version_name: versionName,
   )
 
   slack(
-      message: "Inji [Beta]- #{versionName} is uploaded to Play store. Description : #{PLAY_CONSOLE_RELEASE_DESCRIPTION}.", 
+      message: "Inji - #{INJI_FLAVOR} Wallet- #{versionName} is uploaded to Play store. Description : #{PLAY_CONSOLE_RELEASE_DESCRIPTION}.",
```
Comment (swatigoel) [inline android/fastlane/Fastfile:88]: Can we use app_name created for each flavor in build gradle

File: android/app/build.gradle [hunk 3]
```diff
@@ -95,12 +96,9 @@ android {
     compileSdkVersion rootProject.ext.compileSdkVersion
 
     ext {
-        APP_NAME_RELEASE = "@string/app_name"
-        APP_NAME_MOSIP = "@string/app_name_mosip"
-        APP_NAME_BETA = "@string/app_name_beta"
+        APP_NAME_INJI= "@string/app_name"
```
[the remark this answers is not in the document]
Comment (Adityankannan) [reply android/app/build.gradle]: Has renamed to APP_NAME

File: android/fastlane/Fastfile (hunk 1 above)
Comment (Adityankannan) [reply android/fastlane/Fastfile]: Have used a function to get the app name.

File: android/fastlane/Fastfile (hunk 2 above)
Comment (Adityankannan) [reply android/fastlane/Fastfile]: Have used a function to get the app name.

**Pull request merged**